inc/driver-models/Pin.h

```diff
@@ -60,8 +60,6 @@ DEALINGS IN THE SOFTWARE.
 
 namespace codal
 {
-    using namespace codal;
-
```

Comment:
Compared with header files adding `using namespace codal` globally, this should be safe to remove, as it was enclosed inside the `namespace codal` already, so it didn't have anny effect.